### PR TITLE
Mocma

### PR DESCRIPTION
**Pull request by wadaniel**

### CODE REVIEW

File: source/modules/problem/multiobjectiveoptimization/multiobjectiveoptimization._hpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#ifndef _KORALI_PROBLEM_MULTIOBJECTIVEOPTIMIZATION_HPP_
+#define _KORALI_PROBLEM_MULTIOBJECTIVEOPTIMIZATION_HPP_
+
+#include "modules/problem/problem.hpp"
+
+namespace korali
+{
+namespace problem
+{
+class Multiobjectiveoptimization : public korali::Problem
```
Comment (SergioMartin86) [inline source/modules/problem/multiobjectiveoptimization/multiobjectiveoptimization._hpp:10]: This is not very Koralic. A better approach is
Problem/Optimization/SingleObjective
Problem/Optimization/MultiObjective
However, in this case extensive re-factoring on existing examples/study cases would be necessary to adjust to the new nomenclature.

An even better approach would be:

Have a vector of `objectives` in the existing `Problem/Optimization` where the default is 1 (single objective). Only MO-CMAES (for now) would support objectives.size() > 1.

File: source/modules/solver/solver.config [hunk 2]
```diff
@@ -14,24 +15,31 @@
     "Description": "Determines how many solver generations to run before stopping execution. Execution can be resumed at a later moment."
    }
  ],
-
+ 
  "Internal Settings":
  [
+   {
+    "Name": [ "Variable Count" ],
```
Comment (SergioMartin86) [inline source/modules/solver/solver.config:22]: New attribute for ALL solvers in korali, only used for MO-CMAES? Also never used as far as I've seen. Perhaps you forgot to remove it?

**Review by SergioMartin86 — CHANGES_REQUESTED** [review]
Please address the comment about making the MO problem more koralic.

File: source/modules/solver/solver.config (hunk 2 above)
Comment (wadaniel) [reply source/modules/solver/solver.config:22]: I wish that every solver uses it sooner or later, 30% define a variable like "dimension" and the rest use the _k0->_variables.size() which I find too long in the code

we keep it and I start including it for some solvers?

File: source/modules/problem/multiobjectiveoptimization/multiobjectiveoptimization._hpp (hunk 1 above)
Comment (wadaniel) [reply source/modules/problem/multiobjectiveoptimization/multiobjectiveoptimization._hpp:10]: but a vector of objectives would imply n-times the model evaluations? I would like to keep the F(x) = [bla , blu]

File: source/modules/solver/solver.config (hunk 2 above)
Comment (wadaniel) [reply source/modules/solver/solver.config:22]: done

File: source/modules/problem/multiobjectiveoptimization/multiobjectiveoptimization._hpp (hunk 1 above)
Comment (wadaniel) [reply source/modules/problem/multiobjectiveoptimization/multiobjectiveoptimization._hpp]: removed mo problem, added evaluateMultiple, done

**Review by SergioMartin86 — APPROVED** [review]

**Review by webepasc — APPROVED** [review]

**Pull request merged**